protos/protos.go

```diff
@@ -126,7 +126,7 @@ func (p Protocol) String() string {
 }
 
 type Protocols interface {
-	BpfFilter(with_vlans bool) string
+	BpfFilter(with_vlans bool, with_icmp bool) string
```

Comment:
I updated all my "internal" code to use camelCase. The "external" code which is in other files like `config.go` and `protos.go` I left as is, not to mix both naming conventions in a single file.

I suggest that the switch from under_score to camelCase is done in one go.